api/swagger/swagger.yaml

```diff
@@ -2629,6 +2651,14 @@ paths:
           description: "The project phase of the file."
           required: false
           type: string
+        - name: projectTypes
```

Comment:
Why do Documents need a project type? Isn't this field just meant for projects only?

Just from reading the ticket, it looks like you'd need the projectTypes field in 4 places:
- authenticated (/project) get
- auth'd put 
- auth'd post
- public (/public/project) get

Reply:
Thanks for catching it, just a misplaced field.